pkgs/servers/bililiverecorder/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  makeWrapper,
+  dotnetCorePackages,
+}:
+
+let
+  pname = "bililiverecorder";
```

Comment:
It seems that they prefers to use rec instead of let, but it is probably okay to write like this

Reply:
So I'm going to leave it there and see if anyone else has a comment with it.

Reply:
I prefer `rec` as well, but this is not a blocker for me